core/system/inc/svc_gw.h

```diff
@@ -55,12 +55,14 @@ static inline uint32_t svc_gw_get_dst_fn(TSecGw *svc_pc)
 
 static inline uint8_t svc_gw_get_dst_id(TSecGw *svc_pc)
 {
-    uint32_t box_id = svc_pc->cfg_ptr - __uvisor_config.cfgtbl_ptr_start;
+    uint8_t box_id;
 
-    if(box_id <= 0 || box_id >= g_vmpu_box_count)
```

Comment:
I've introduced the additional check on `!box_id` to keep consistency with the previous functionality. A comment explains that. I'd still like to know if we want to allow sgw to box 0 -- in that case I'd introduce a new commit with the change.